jplag/src/main/java/de/jplag/SubmissionSetBuilder.java

```diff
@@ -216,7 +212,7 @@ private Submission loadBaseCodeViaName(String baseCodeName, File rootDirectory,
             throw new BasecodeException(String.format("Basecode path \"%s\" relative to the working directory could not be found.", baseCodeName));
         } else {
             // Found a base code as a submission, report about legacy usage.
-            System.out.println("Legacy use of the -bc option found, please specify the basecode by path instead of by name!");
+            logger.info("Legacy use of the -bc option found, please specify the basecode by path instead of by name!");
```

Comment:
We should make this a warning, as we might not support the legacy backward compatibility forever.